packages/apps/docs/src/components/PageHelpful/styles.css.ts

```diff
@@ -23,7 +23,7 @@ export const textAreaClass = style([
     '::placeholder': {
       color: vars.colors.$gray40,
     },
-    [`${darkThemeClass} &::placeholder`]: {
+    [`.${darkThemeClass} &::placeholder`]: {
```

Comment:
this is not needed.

Reply:
The class name variable does not and should not contain a dot (a selector might contain one) and the stylesheet parser in our tests complains about it. Maybe there's something that fixes it in the chain eventually, but when parsing the stylesheet happy-dom rightfully complained about it.

Reply:
Well, this is embarrassing. I've been analyzing stylesheets spitted out by the stylesheet parser from happy-dom, and found those dots missing. Now I can no longer reproduce this, so the dots are back. Apologies!

Reply:
Yeah, the variable shouldn't include the dot, so adding the dot is necessary

Reply:
So this PR no longer has changes related to this, and the migrated tests are passing, so let's leave it at this for now.